README.md

```diff
@@ -15,18 +15,37 @@ level of reference detail, i.e. only references for actually used
 functionality will be presented back if software provides multiple
 citeable implementations.
 
-![Example](examples/duecredit_example.gif)
+## Installation
+
+Duecredit is easy to install via pip, simply type:
+ 
+ `pip install duecredit`
+
+## Examples
 
-duecredit 101
-=============
+### To cite the modules you are using 
```

Comment:
it's more methods than modules, because we can wrap also individual functions...how about `modules --> methods`?

Reply:
I would say modules and methods